public/app/features/transformers/extractFields/ExtractFieldsTransformerEditor.tsx

```diff
@@ -96,6 +105,13 @@ export const extractFieldsTransformerEditor = ({
           />
         </InlineField>
       </InlineFieldRow>
+      {options.format === 'regexp' && (
```

Comment:
```suggestion
      {options.format === FieldExtractorID.RegExp && (
```

Reply:
Could update the other ones in this file too, I suppose.

Reply:
yeah, didn't want to grow the changeset more than strictly needed, but sounds fine.